plink-front/src/app/components/coin/coin.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="coin-container">
+  <p class="coin-name">{{coin.name}}</p>
+  <p class="coin-price">{{coin.price | currency}}</p>
+  <div class="coin-crypt">
+    <span *ngIf="crypto; else noCrypto" class="green-cont"><div class="led-green"></div>Si</span>
```

Comment:
delete the led-green div since you replaced it for a `::before` pseudoelement

Reply:
I think you could use an ngClass and a conditional operator instead of an ngTemplate